### PR TITLE
Add neurotic recipe

### PR DESCRIPTION
**Pull request by jpgill86**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example) (found in tarball)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jpgill86** [comment]
Ready to merge!

File: recipes/neurotic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    - pip
+    - python >=3.6
+  run:
+    - av
+    - ephyviewer >=1.2.0
+    - numpy
+    - packaging
+    - pandas
+    - pyqt
+    - python >=3.6,<3.8  # <3.8 required for pyqtgraph 0.10.0
```
Comment (beckermr) [inline recipes/neurotic/meta.yaml]: I don't see pyqtgraph in the dependencies.

File: recipes/neurotic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+{% set version = "1.1.1" %}
+{% set sha256 = "1dceebe21845828b6783721a82bfd61cd2f87fcca6e373db7ec1555e51ce138e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (beckermr) [inline recipes/neurotic/meta.yaml]: go ahead and put the sha here directly.

File: recipes/neurotic/meta.yaml (hunk 1 above)
Comment (jpgill86) [reply recipes/neurotic/meta.yaml]: pyqtgraph is required only indirectly, through ephyviewer. The pyqtgraph recipe should have the Python<3.8 restriction built in, but it doesn't. I've already opened a PR (conda-forge/pyqtgraph-feedstock#11) to fix that, and if it's accepted the explicit restriction to <3.8 won't be needed here anymore. For now, I need it.

Comment (beckermr) [reply recipes/neurotic/meta.yaml]: Ahhh ok.

File: recipes/neurotic/meta.yaml (hunk 2 above)
Comment (jpgill86) [reply recipes/neurotic/meta.yaml]: Comin' right up!

**Comment by jpgill86** [comment]
Thank you @beckermr!

**Pull request merged**